src/applications/financial-status-report/sass/financial-status-report.scss

```diff
@@ -373,8 +378,10 @@ $chapters: veteranInformationChapter, householdIncomeChapter,
 .va-button-override va-button[text="Continue to review page"] {
   &::part(button) {
     &::after {
-      content: "\f101";
+      content: "";
       margin-left: 8px;
```

Comment:
The `margin-left` should be removed now as it adds too much extra spacing:

![image](https://github.com/user-attachments/assets/78129c02-4424-4cdf-9629-565250cb8507)